backend/question-service/lambda/app/scraper/handler.py

```diff
@@ -0,0 +1,189 @@
+import time
```

Comment:
can help to add a `.env.example` in this directory? so that anyone who needs to run it locally for any reason knows the env vars that need to be provided

Reply:
Resolved in https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g03/commit/de1eb1590e1c47555ff72b5033e505e90f480449:
- Added .env.example file for necessary environment variables to run the function locally